charts/dapr/charts/dapr_config/values.yaml

```diff
@@ -1,4 +1,6 @@
 mtls:
   enabled: "true"
   workloadCertTTL: 24h
-  allowedClockSkew: 15m
\ No newline at end of file
+  allowedClockSkew: 15m
+
+  enabled: true
```

Comment:
Can we rename `enabled` to be more meaningful ? e.g. `config_chart_included`

And please add the purpose of this value as a comment.